src/Symfony/Component/HttpKernel/EventListener/DebugHandlersListener.php

```diff
@@ -104,12 +104,11 @@ public function configure(Event $event = null)
             if ($event instanceof KernelEvent) {
                 if (method_exists($kernel = $event->getKernel(), 'terminateWithException')) {
                     $request = $event->getRequest();
-                    $hasRun = &$this->hasTerminatedWithException;
-                    $this->exceptionHandler = function (\Exception $e) use ($kernel, $request, &$hasRun) {
-                        if ($hasRun) {
+                    $this->exceptionHandler = function (\Exception $e) use ($kernel, $request) {
```

Comment:
Another option here is to make the closure `static`. Thos would have the benefit of removing a circular reference.

Reply:
I like the proposal, would you consider enabling `StaticLambdaFixer` CS fixer rule (I have to revert this specific change then)?

Reply:
please revert and add `static` instead yes
dunno about `StaticLambdaFixer` - why not :)

Reply:
Will do + will create a PR for static closures.

Reply:
BTW which branch to target with static closures, 3.4?

Reply:
> BTW which branch to target with static closures, 3.4?
maybe none: the rule is risky and I'm not sure it makes sense to apply it globally now that I reviewed what it does :)

Reply:
Got it, but at least you are now aware of it =)